src/extensions/custom-view/bootstrap-table-custom-view.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @author: Dustin Utecht
+ */
+
+const Utils = $.fn.bootstrapTable.utils
+
+$.extend($.fn.bootstrapTable.defaults, {
+  customView: false,
```

Comment:
Is this the function that will return the view, right? If yes, please let's init it with `$.noop` or `function return false`. This is to be more explicit

Reply:
Do you mean e.g.
```
$.extend($.fn.bootstrapTable.defaults, {
  customView: function () {
    return false
  },
  showCustomView: false,
  showCustomViewButton: false
})
```

Then check on the code if `const value = Utils.calculateObjectValue(this, this.options.customView, [data], '')` dont return false ?

Reply:
You said in a comment that is `value` is null or empty an text "null" will be displayed which is great for me.